codesign/inputparse.go

```diff
@@ -61,6 +75,46 @@ func ParseConfig(input Input, cmdFactory command.Factory) (Config, error) {
 	}, nil
 }
 
+// parseConnectionOverrideConfig validates and parses the step input-level connection parameters
+func parseConnectionOverrideConfig(keyPathOrURL stepconf.Secret, keyID, keyIssuerID string, logger log.Logger) (*devportalservice.APIKeyConnection, error) {
+	var key []byte
+	if strings.HasPrefix(string(keyPathOrURL), "https://") {
+		resp, err := retryhttp.NewClient(logger).Get(string(keyPathOrURL))
+		if err != nil {
+			return nil, fmt.Errorf("API key download error: %s", err)
+		}
+
+		defer func(Body io.ReadCloser) {
+			err := Body.Close()
+			if err != nil {
+				logger.Errorf(err.Error())
+			}
+		}(resp.Body)
+		if resp.StatusCode != http.StatusOK {
+			return nil, fmt.Errorf("API key HTTP response %d: %s", resp.StatusCode, resp.Body)
+		}
+
+		key, err = io.ReadAll(resp.Body)
+		if err != nil {
+			return nil, err
+		}
+	} else {
+		var err error
```

Comment:
For backward compatibility and consistency, can we make sure to optionally also support the `file://` prefix (used for example at https://github.com/bitrise-steplib/steps-ios-auto-provision-appstoreconnect/blob/33810ffbcdc37b6155e7ebd7c4f772d18b09d6df/step.yml#L99)